triton-server/src/download/actions.ts

```diff
@@ -263,5 +278,5 @@ export function setDatasetDownloading(datasetID: number, value: boolean) {
 // Helpers
 
 function currentDateToString() {
-    return new Date().toISOString()
+    return dayjs().format()
```

Comment:
the base object does the job just fine

Reply:
hard disagree, `toISOString` only returns UTC time which is problematic since we're not in UTC.